doc/reference/reference_lua/msgpack.rst

```diff
@@ -526,3 +564,220 @@ and :ref:`YAML <yaml-cfg>`.
         - [20, null, 20]
         ...
 
+..  _msgpack-object:
+
+..  function:: object(lua_value)
+
+    Since version 2.10.0.
+
+    Encode an arbitrary Lua object into the MsgPack format.
+
+    :param lua-object lua_value: a Lua object of any type.
+
+    :return: encoded MsgPack data encapsulated in a MsgPack object.
+
+    :rtype: userdata
+
+    **Example:**
+
+    ..  code-block:: lua
+
+        local msgpack = require('msgpack')
+        -- Create a MsgPack object from a Lua object of any type
+        mp = msgpack.object(123)
```

Comment:
Nit: in all other examples, you define all variables as local. You could add `local` here as well.